src/arrayMethodSort.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compare = compareFunction
+    || ((a, b) => a.toString() > b.toString() ? 1 : -1);
```

Comment:
```suggestion
    || ((a, b) => Number(a.toString() > b.toString()));
```
`False` -> 0
`True` -> 1